src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -237,6 +237,29 @@ public override void VisitNamedType(NamedTypeSymbol symbol)
         }
 
 #nullable enable
+        public override void VisitMethod(MethodSymbol symbol)
+        {
+            if (symbol is SourceExtensionImplementationMethodSymbol implementation)
+            {
+                var symbolForDocComment = implementation.UnderlyingMethod is SourcePropertyAccessorSymbol accessorSymbol
```

Comment:
>implementation.UnderlyingMethod is SourcePropertyAccessorSymbol [](http://example.com/codeflow?start=42&length=63)

Consider using ```IsAccessor``` helper instead. The type check won't work for events, for example.